.gitignore

```diff
@@ -1,9 +1,8 @@
 .vs/
-bin/
-obj/
```

Comment:
Is it really a good idea to remove these from gitignore? I get that we're not using these folders as part of the official build, but this just seems like it could cause annoyances.

Reply:
I removed this intentionally because I was hoping it would elevate the visibility of problems with misconfigured obj and bin paths.

Reply:
Also, if it wasn't clear from the PR description, the obj and bin folders share a common root under `artifacts/`

Reply:
It does seem clear, I just expect that we're going to find a case in the future where this happens and is-annoying.